lookup_plugins/retrieve_conjur_variable.py

```diff
@@ -2,9 +2,11 @@
 
 import os.path
 import ssl
+import yaml
```

Comment:
This is `pyyaml`, which will fail because it most likely won't be on an end user's remote.  We'd strongly prefer not to require people to install `pyyaml` as a requirement. Can we just parse the value out of the file, or find a way to include it with the module?

Reply:
Leftover from merge, will be deleted